content/en/account_management/rbac/data_access.md

```diff
@@ -0,0 +1,183 @@
+
+### Tag incoming data
+
+Data Access Control relies on tags and attributes in your data that can be used to define an access boundary. If you do not have tags defined, consider [Getting Started with Tags][4] before moving on to configure Data Access Control.
+
+## Configure data access
+
+Data Access Control lets you create a restricted dataset, which defines a set of data which only users in specified teams or roles can access.
+
+### Datadog site
+
+Log in as a user assigned the Datadog Admin role, or any user with a role in your organization with the [`user_access_manage` permission][5].
+
+1. Navigate to [Organization Settings][6].
+1. On the left side of the page, select [Data Access Controls][7].
+1. Click **New Restricted Dataset**.
+
+In order to create a restricted dataset, you need to identify the data to be restricted with a query.
+
+{{< img src="/account_management/rbac/restricted_dataset.png" alt="Create a restricted dataset dialog. Selects data in RUM, APM, Logs, and Metrics matching the tag service:hr. Grants access to a Privileged access team.">}}
+
+Name Dataset
+: A descriptive name to help users understand what data is contained in the dataset. 
+
+Select data to be included in this Dataset
+: The boundary definition that describes which data to restrict to a specific set of users. Boundaries are query statements with limitations that allow an access manager to define the scope of sensitive data to be protected. The supported telemetry types are custom metrics, RUM, APM traces, and logs
+
+Grant access
+: Select one or more teams or roles that may access the content bound in the Restricted Dataset. Any users who are not members of these groups are blocked from accessing this data.
+
+You may create a maximum of 10 key:value pairs per restricted dataset. Consider defining an additional restricted dataset if you need additional pairs.
+
+After you define the dataset by filling in all of the fields, click **Create Restricted Dataset** to apply the restricted dataset to your organization.
+
+You may create a maximum of 100 restricted datasets. If you need a higher limit, reach out to Support.
+
+### API
+The Data Access Control API is under development and should be considered unstable. Future versions may be backward incompatible. 
+
+Terraform support will be announced after Data Access Control is generally available.
+
+## Select tags for access
+
+Each restricted dataset can control access to multiple types of data, such as metrics. You are free to use the same or different tags across multiple types of telemetry. Within each telemetry type, you must use a _single_ tag or attribute to define your access strategy.
+
+If you have too many combinations of tags or attributes to fit within these constraints, consider [revisiting your tagging][4] to define a new tag that better reflects your access strategy.
+
```

Comment:
```suggestion
# Supported examples and restrictions for Restricted Datasets
```
I think the flow and formatting can be improved here but feel free to take or reject the suggestion